From 68b1e8904d80c018179961fe9322727f3f91d936 Mon Sep 17 00:00:00 2001
From: Zhu Yuqing <b48954@freescale.com>
Date: Thu, 22 May 2014 11:05:16 +0800
Subject: [PATCH 67/70] ENGR00314360 fix UI: The OTA progress bar was shrank
 to the right bottom part of dialogue

change the layout_gravity of the progress bar from "right" to "fill"

delete some redundant codes

Signed-off-by: Zhu Yuqing <b48954@freescale.com>
---
 FSLOta/res/layout/main.xml | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/FSLOta/res/layout/main.xml b/FSLOta/res/layout/main.xml
index 592a258..7de1107 100644
--- a/FSLOta/res/layout/main.xml
+++ b/FSLOta/res/layout/main.xml
@@ -49,7 +49,7 @@
             android:layout_height="wrap_content"
             android:layout_column="1"
             android:layout_columnSpan="5"
-            android:layout_gravity="right"
+            android:layout_gravity="fill"
             android:layout_row="3"
             android:layout_rowSpan="3" />
 
@@ -90,9 +90,4 @@
             android:layout_row="4" />
     </GridLayout>
 
-        android:id="@+id/linearLayout1"
-        android:layout_width="318dp"
-        android:layout_height="match_parent"
-        android:layout_gravity="center_horizontal"
-        android:orientation="vertical" >
 </LinearLayout>
-- 
1.8.0

